presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -19,6 +19,7 @@
     private static final String OPTIMIZE_HASH_GENERATION = "optimize_hash_generation";
     private static final String DISTRIBUTED_JOIN = "distributed_join";
     private static final String HASH_PARTITION_COUNT = "hash_partition_count";
+    private static final String TASK_PARALLEL_COUNT = "experimental_task_parallel_count";
```

Comment:
I think we should just name it `task_parallel_count`. All these session properties are basically experimental.